### PR TITLE
Update LibraryEvolution.rst for new attribute names and @_alwaysEmitIntoClient

### PR DESCRIPTION
**Pull request by slavapestov**

### CODE REVIEW

**Comment by slavapestov** [comment]
@swift-ci Please smoke test

File: docs/LibraryEvolution.rst [hunk 1]
```diff
@@ -408,12 +409,28 @@ bodies of inlinable functions have the following restrictions:
   guards.
 
 
+Always Emit Into Client
+-----------------------
+
+A function, property or subscript annotated as ``@_alwaysEmitIntoClient``
```
Comment (jrose-apple) [inline docs/LibraryEvolution.rst]: Nitpick: "computed property".

**Review by jrose-apple — APPROVED** [review]
Thank you!

**Comment by slavapestov** [comment]
@swift-ci Please smoke test

**Pull request merged**